README.md

```diff
@@ -25,60 +34,55 @@ Kuzzle enables you to build modern web applications and complex IoT networks in
 The easiest way to setup a kuzzle server for Linux-like systems without prerequisites is to download and run our installation script:
 
 ```bash
-$ sudo bash -c "$(curl http://get.kuzzle.io/)"
+$ sudo bash -c "$(curl https://get.kuzzle.io/)"
 ```
 
-You can get detailed information about how to [start kuzzle with docker on docs.kuzzle.io](https://docs.kuzzle.io/guide/essentials/installing-kuzzle/#docker)
+You can get detailed information about how to [start kuzzle with docker on docs.kuzzle.io](https://docs.kuzzle.io/guide/1/essentials/installing-kuzzle/#docker)
 
 ### Manual install
 
-Check our [complete installation guide on docs.kuzzle.io](https://docs.kuzzle.io/guide/essentials/installing-kuzzle/#manually)
+Check our [complete installation guide on docs.kuzzle.io](https://docs.kuzzle.io/guide/1/essentials/installing-kuzzle/#manual-installation)
 
 ## Quick start with Kuzzle
 
-* [Install and start Kuzzle server](https://docs.kuzzle.io/guide/essentials/installing-kuzzle/)
-* [Choose a SDK](https://docs.kuzzle.io/sdk-reference/essentials/)
+* [Install and start Kuzzle server](https://docs.kuzzle.io/guide/1/essentials/installing-kuzzle)
+* [Choose a SDK](https://docs.kuzzle.io/sdk-reference/)
 * Build your application without caring about your backend !
 
-Check the [**Getting started page on docs.kuzzle.io**](https://docs.kuzzle.io/guide/getting-started/)
+Check the [**Getting started page on docs.kuzzle.io**](https://docs.kuzzle.io/guide/1/getting-started/first-steps/)
 
-### NodeJS Sample
+### Node.js samples
```

Comment:
```suggestion
### Node.js examples
```

Reply:
Also, after reading that code example, I'm convinced that it's more confusing than anything: that example subscribes to database changes, and then... nothing. It doesn't show anything useful by itself.

My take on this is that we should delete that section, especially if you apply my proposal about the documentation links list below